en/administration/configuration/database/oracle.md

```diff
@@ -11,11 +11,11 @@ http://support.rundeck.com/customer/en/portal/articles/2415681-oracle-setup)
 
 [http://www.oracle.com/technetwork/database/features/jdbc/default-2280470.html](http://www.oracle.com/technetwork/database/features/jdbc/default-2280470.html)
 
-* Copy the downloaded file `ojdbc7.jar` to the `$RDECK_BASE/server/lib` for war launcher or in /var/lib/rundeck/lib (create it) for RPM and DEB installations
-* Update `$RDECK_BASE/etc/rundeck-config.properties`:
+* Copy the downloaded file `ojdbc7.jar` to the `$RDECK_BASE/server/lib` for war launcher or in `/var/lib/rundeck/lib` (create it) for RPM and DEB installations
+* Update `$RDECK_BASE/server/config/rundeck-config.properties`:
```

Comment:
This location is different for rpm/deb vs war install, so I think it would be better to link to the Configuration Layout section here https://rundeck.org/docs/administration/configuration/configuration-file-reference.html#configuration-layout

Reply:
I have edited my repo and commit it to add the configuration layout, please check if it is all good now.